metaflow/plugins/cards/card_cli.py

```diff
@@ -454,7 +455,11 @@ def _add_token_html(html):
     def _add_token_json(json_msg):
         if json_msg is None:
             return None
-        return {"reload_token": _reload_token(), "data": json_msg}
+        return {
+            "reload_token": _reload_token(),
+            "data": json_msg,
+            "created_on": time.time(),
```

Comment:
driveby change: add a timestamp in the dataupdate call. It helps discard data updates that maybe stale.